.gitignore

```diff
@@ -25,3 +25,26 @@
 /test/test_config.ini
 # Schema files needed for testing
 /test/adapter/schemas
+
+
+# Python build artifacts
+__pycache__/
+*.py[cod]
+*$py.class
+*.so
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+*.egg-info/
+.installed.cfg
+*.egg
```

Comment:
Those are all common Python build artifacts. It would be cleaner to only mention those that actually get created with our build process, which depends on how we actually end up building the module.